package.json

```diff
@@ -26,7 +26,7 @@
     "eslint": "^7.32.0",
     "ethereum-waffle": "^3.4.0",
     "ethers": "^5.4.4",
-    "hardhat": "^2.10.2",
+    "hardhat": "^2.12.2",
```

Comment:
Forking Arbitrum for testing fails with `hardhat@2.10.2` from version `hardhat@2.12.2` a fix has been added